docs/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ Version 3.43.0 (?? ??, 2024)
 
 **User-visible changes:**
 
+Renamed command-line arguments:
+ * `-AskipDirs` has been renamed to `-AskipFiles`.
+   `-AskipDirs` will continue to work for the time being.
```

Comment:
Is there a need for an explicit `-AskipDirs` command? I remember implementing it [here](Vhttps://github.com/typetools/checker-framework/pull/6385) for #6093

Reply:
Sorry, I don't understand your question.  The `-AskipDirs` command-line option is useful and continues to work, but users will be encouraged to use `-AskipFiles` (with identical semantics) instead.  This pull request added `-AonlyFiles` which is new.

Reply:
Sorry, I was being unclear.

I think I was confused with the re-name from `-AskipDirs` to `-AskipFiles`. I know there isn't much of a distinction between files/directories on Unix systems so it makes sense there. Is this the same for non-Unix systems like Windows?